7-beta/Dockerfile

```diff
@@ -54,6 +54,8 @@ RUN curl -o phantomjs.tar.bz2 -SL https://bitbucket.org/ariya/phantomjs/download
 RUN curl -sS https://getcomposer.org/installer \
     | php -- --version=1.10.17 --install-dir=/usr/local/bin --filename=composer
 
+RUN apt update ; apt install -qy node-less ; rm -rf /var/lib/apt/* /var/cache/apt/*
```

Comment:
Please add `node-less` to the list of apt dependencies on line 5 instead.

Reply:
Sure, thank you. I must have missed that somehow.
I also added, that the apt cache will be removed as well